### DISCUSSION TITLE
Almacenar Datos

### DISCUSSION THREAD

**Discussion by jricardocam** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#500] [Q&A]
Hola! 
Ya logro mostrar los datos del formulario mientras se llenan los campos, el problema es que al dar clic en el boton todos los datos se borran y no se guarda nada.
comparto mi codigo
![Formulario](https://user-images.githubusercontent.com/99063528/159777218-bde49ff8-b857-4143-ab12-d882bad9c2fd.png)
![Pastelero](https://user-images.githubusercontent.com/99063528/159777220-3002ef70-fba3-4c9e-a968-1ae09bf501bb.png)
![Store](https://user-images.githubusercontent.com/99063528/159777225-1ff5cb33-e240-4187-bb4f-df4e5cb59d45.png)

**Comment by doguedogue** [comment]
https://web.microsoftstream.com/user/96eeb66f-b984-4083-8ea6-74c34a07a34a
